Amq_List_Broker-main/template/newcloudformationforlambda.yaml

```diff
@@ -0,0 +1,99 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "get activemq each topic count"
+Parameters:
+  LambdaVersion:
+    Description: "LambdaFunction Version"
+    Type: String
+    Default: ""
+  brokerUserName:
```

Comment:
Parameter names in camelcase to be consistent.. brokerUserName=> BrokerUserName, clientname => ClientName